### PR TITLE
[WIP] DO ALL TEH DOCS

### PR DESCRIPTION
**Pull request by hamhands**
### To do:
- [x] admin
- [x] conductor
  - [x] manager
    - [x] views
  - [x] metrics
    - [x] views
  - [x] upload
    - [x] views
  - [x] forms
  - [x] util
  - [x] validators
- [x] data
  - [x] importer
    - [x] costars
    - [x] importer
    - [x] nigp
    - [x] old_contracts
    - [x] scrape_county
    - [x] stages_and_flows
    - [x] state
  - [x] companies
  - [x] contract_stages
  - [x] contracts
  - [x] flows
  - [x] models
  - [x] searches
  - [x] stages
- [x] jobs
  - [x] beacon_nightly
  - [x] job_base
  - [x] scout_nightly
- [x] opportunities
  - [x] admin
    - [x] views
  - [x] front
    - [x] views
  - [x] forms
  - [x] models
  - [x] util
  - [x] validators
- [x] public
  - [x] models
  - [x] views
- [x] scout
  - [x] forms
  - [x] util
  - [x] views
- [x] users
  - [x] forms
  - [x] models
  - [x] views
### Current status:

giphy 26

### CODE REVIEW

**Comment by bsmithgall** [comment]
[Docs-dev dos available here](http://pittsburgh-purchasing.readthedocs.org/en/docs-dev/)

File: docs/source/public.rst [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Public & users
 ==============
 
-Models
-------
+Models used
+-----------
 
 .. automodule:: purchasing.users.models
```
Comment (bsmithgall) [inline docs/source/public.rst]: This should be switched to :py:class: reference tags, like in the Beacon or Scout docs

File: purchasing/users/forms.py [hunk 2]
```diff
@@ -6,6 +6,13 @@
 from purchasing.users.models import Department
 
 class DepartmentForm(Form):
+    '''Allows user to update profile information
+
+    :var department: sets user department based on query displaying all departments,
```
Comment (bsmithgall) [inline purchasing/users/forms.py]: What?

File: purchasing_test/integration/users/test_users.py [hunk 3]
```diff
@@ -19,13 +19,6 @@ def setUp(self):
         DepartmentFactory.create(name='New User').save()
         self.department1 = DepartmentFactory.create(name='Test').save()
 
-    def test_login_route(self):
```
Comment (bsmithgall) [inline purchasing_test/integration/users/test_users.py:24]: :bomb: !

**Comment by tmaybe** [comment]
I tried following the [installation instructions](https://github.com/codeforamerica/pittsburgh-purchasing-suite/blob/4b62f0be508bf86b73019856a969d6b68caee908/docs/source/develop.rst#quick-local-installation-using-make) and hit a couple snags:
1. I think you should say explicitly that the developer needs to change into the repo directory, as you do in the _More detailed installation instructions_.
2. I think you should also say explicitly that the developer should `pip install -r requirements.txt`, even though that instruction's contained in [the linked how-to](https://github.com/codeforamerica/howto/blob/master/Python-Virtualenv.md).
3. Running `make setup` as instructed fails because Sphinx is required but isn't in `requirements.txt`. I installed it manually.
4. After that, the make started, but failed at `bower install` with this error:

```
bower install
make[1]: bower: No such file or directory
make[1]: *** [install] Error 1
make: *** [setup] Error 2
```

I didn't try to diagnose that one.

**Comment by bsmithgall** [comment]
@tmaybe Thanks! 1-3 should all be fixed now. I added 1, and 2-3 can now be handled via `make setup` as before. There was an issue with importing the sphinx makefile where it was executing some bash on every call into the makefile. I moved that into a function and am calling the function on any make script that involves sphinx instead.

With 4, it's a bit tricker. The [bower home page](http://bower.io/) says that you need to use `npm install -g`, which involves a global install. I will investigate on a clean install if we can get away with just installing bower into the local dependencies via `node_modules`.

**Comment by bsmithgall** [comment]
This is ready to go I think!

**Comment by msknee** [comment]
:+1:

**Pull request merged**